README.md

```diff
@@ -18,3 +18,6 @@ If you are creating an environment *derived* from this one, you might also want
 If you are creating a new folder for a new image, please use a meaningful prefix to the folder name; for example, use a prefix of 'gdal' for an image that has gdal pre-installed.
 
 Also, be sure to read about how to [contribute](https://github.com/AgPipeline/Organization-info) to this organization.
+
+## Testing 
+The testing modules and readme can be found in the [testing image](https://github.com/AgPipeline/ua-gantry-environment/tree/test-development/test-image)
```

Comment:
Need to append ' folder' to the end of the line?

Reply:
Yup, you're right.